Lib/test/test_imaplib.py

```diff
@@ -468,16 +468,6 @@ def test_logincapa(self):
             _server = self.imap_class(self.host, self.port)
             self.check_logincapa(_server)
 
-    def test_logincapa_with_client_certfile(self):
```

Comment:
Okay.  Why is this bugfix part of the "Backport CI config from master" PR?  As far as I can tell, it has nothing to do with backporting the config for AppVeyor, CodeConv, and Travis.

Unless this bugfix is necessary for running AppVeyor, CodeConv, or Travis, I would like this change removed from the PR.